website/ibm.erb

```diff
@@ -68,6 +68,9 @@
         <li<%= sidebar_current("docs-ibm-datasource-cis") %>>
           <a href="#">Cloud Internet Services Data Sources</a>
           <ul class="nav nav-visible">
+            <li<%= sidebar_current("docs-ibm-datasource-cis-dns-record") %>>
+              <a href="/docs/providers/ibm/d/cis_dns_record.html">cis</a>
```

Comment:
replace `docs-ibm-datasource-cis-dns-record` to `docs-ibm-datasource-cis-dns-records`
`/docs/providers/ibm/d/cis_dns_record.html` to `/docs/providers/ibm/d/cis_dns_records.html`

Reply:
done